lib/customer.rb

```diff
@@ -0,0 +1,33 @@
+require 'csv'
+# require 'awesome_print'
+
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
Do these need to be accessors?  Since the user of your class doesn't need the ability to directly change the email and address attributes, you can instead make these attr_reader instead.